Diary/Sources/Controllers/DetailDiaryViewController.swift

```diff
@@ -0,0 +1,100 @@
+//
+//  DetailDiaryViewController.swift
+//  Diary
+//
+//  Created by yeton, hyeon2 on 2022/08/18.
+//
+
+import UIKit
+
+final class DetailDiaryViewController: UIViewController {
+    private var content: DiarySample?
+    
+    let textView: UITextView = {
+        let textView = UITextView()
+        textView.font = UIFont.preferredFont(forTextStyle: .body)
+        textView.translatesAutoresizingMaskIntoConstraints = false
+
+        return textView
+    }()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        configureNavigationItemTitle()
+        configureAttributes()
+        configureLayout()
+        configureTextView()
+        registerForKeyboardNotification()
+    }
+    
+    private func configureNavigationItemTitle() {
+        let time = content?.createdAt ?? Date().timeIntervalSince1970
+        let date = Date(timeIntervalSince1970: time)
+        
+        navigationItem.title = DateFormatter().format(data: date)
+    }
+    
+    private func configureAttributes() {
+        view.addSubview(textView)
+        view.backgroundColor = .white
+    }
+    
+    private func configureLayout() {
+        NSLayoutConstraint.activate([
+            textView.topAnchor.constraint(
+                equalTo: view.safeAreaLayoutGuide.topAnchor
+            ),
+            textView.bottomAnchor.constraint(
+                equalTo: view.safeAreaLayoutGuide.bottomAnchor
+            ),
+            textView.leadingAnchor.constraint(
+                equalTo: view.safeAreaLayoutGuide.leadingAnchor
+            ),
+            textView.trailingAnchor.constraint(
+                equalTo: view.safeAreaLayoutGuide.trailingAnchor
+            )
+        ])
+    }
+    
+    private func configureTextView() {
+        guard let content = content else {
+            return
+        }
+
+        textView.text = content.title + "\n\n" + content.body
+        textView.contentOffset.y = 0
+    }
+
+    private func registerForKeyboardNotification() {
+        NotificationCenter.default.addObserver(
+            self,
+            selector: #selector(keyBoardShow),
+            name: UIResponder.keyboardWillShowNotification,
+            object: nil
+        )
+    }
```

Comment:
[removeObserver(_:name:object:)](https://developer.apple.com/documentation/foundation/notificationcenter/1407263-removeobserver) 로 등록된 Observer를 지워주려고 했지만, 
공식문서에 `iOS 9.0 이상`이고 `addObserver(_:selector:name:object:)`메서드를 사용한 경우에는 관찰자 등록을 취소할 필요가 없다고 나와있었습니다. (아래 해당 부분 첨부해놨습니다!)

>If your app targets iOS 9.0 and later or macOS 10.11 and later, and you used [addObserver(_:selector:name:object:)](https://developer.apple.com/documentation/foundation/notificationcenter/1415360-addobserver) to create your observer, you do not need to unregister the observer. If you forget or are unable to remove the observer, the system cleans up the next time it would have posted to it.

저희 앱은 현재 `iOS 14.0`이고, `addObserver(_:selector:name:object:)` 사용하고 있기에 위의 근거로 별도의 제거 코드를 작성하지 않습니다☺️